pc-air/pc-air-local.json

```diff
@@ -0,0 +1,29 @@
+{  
+  "pcair.kinship_file": "/Users/julianlucas/Desktop/github/analysis_pipeline_WDL/_test-data-and-truths_/pc-air/workflow_inputs/1KG_king_ibdseg_Matrix.RData",
```

Comment:
The local testdatas are all supposed to run from the top folder of the repo itself so that users who aren't named julianlucas can just plug and play. Everything else looks good on this file, just change all the paths so they start with _test-data-and-truths_.

Reply:
Fair enough.